packages/client/package.json

```diff
@@ -55,7 +54,6 @@
     "jest-matcher-specific-error": "^1.0.0",
     "lint-staged": "^13.0.0",
     "nock": "^13.1.3",
-    "node-fetch": "^2.6.7",
```

Comment:
Should also remove `@peculiar/webcrypto`

Reply:
updated